rakelib/vendor.rake

```diff
@@ -124,6 +124,33 @@ namespace "vendor" do
   end # task gems
   task "all" => "gems"
 
+  task "deps", [:gemfile] => [ "dependency:bundler" ] do |task, args|
+    dependencies = {}
+    # grab the development dependencies
+    gem_home = LogStash::Environment.logstash_gem_home
+    Dir.glob("#{gem_home}/logstash-*/*.gemspec") do |gemspec|
+      spec = Gem::Specification.load(gemspec)
+      spec.development_dependencies.each do |dependency|
+        dependencies[dependency.name] = dependency
+      end
+    end
+    deps_gemfile = args[:gemfile]
+    temp_data    = File.read(deps_gemfile)
+    # generate the gemfile.
+    File.open(deps_gemfile, "a") do |file|
+      dependencies.values.each do |dependency|
+        next if dependency.name.start_with?('logstash-')
```

Comment:
When we run the tests for the plugins inside logstash are we targeting the logstash gems only?
If we install all the dependencies for third party gems I doubt we will be able to run the tests.